File: pyStudy/day1230/Study10.java
package day1230;

public class Study10 {
    public static void main(String[] args) {
        int num1 = 1;
        int num2 = 3;
        
        int s = mySum(num1,num2);

        System.out.println(s);
        sayHi();
    }

    // static : 정적변수
    static int mySum(int x, int y){
        int result = x + y;

        return result;
    }

    static void sayHi(){
        System.out.println("Hi~!");
    }

}

File: pyStudy/day1230/study10.py
# 함수

def mySum(x, y):
    """이 함수는 더하는 함수다."""  # doc 스트링 : 함수 설명서
    result = x + y;
    return result;

def returnHi():
    return "Hi~";


def sayHi():
    print("Hi~!");

num1 = 1;
num2 = 3;

s = mySum(num1, num2);
print(s);
sayHi();
print(returnHi());

print(mySum.__doc__);